cirq/ops/common_gates.py

```diff
@@ -820,6 +866,29 @@ def _pauli_expansion_(self) -> value.LinearDict[str]:
             'ZX': global_phase * -c,
         })
 
+    def controlled(self,
+                   num_controls: int = None,
+                   control_values: Optional[Sequence[
+                       Union[int, Collection[int]]]] = None,
+                   control_qid_shape: Optional[Tuple[int, ...]] = None
+                  ) -> raw_types.Gate:
+        """
+        Specialize controlled for CNotPow to return corresponding controlled
+        CCXPow when the last control (which acts first semantically) is a
```

Comment:
I think it might be worth elaborating why a controlled-CNOT is a CCX gate here.  I'm not sure that's immediately obvious to everyone.  I had to think about it for a minute anyway.  The other examples that you added are more straightforward (though it might not hurt to add extra explanation), but this one seems less obvious.

Reply:
My understanding is that by definition CCX means controlled-controlled-X and CNOT means controlled-X so I am not sure how to elaborate on this in the pydoc.

Reply:
Now addressed in the pydoc